packages/ui/src/components/sidebar/sidebar.tsx

```diff
@@ -123,6 +123,8 @@ const SidebarProvider = forwardRef<
   useEffect(() => {
     if (isMobile) return
 
+    document.body.style.setProperty('--sidebar-width-icon', SIDEBAR_WIDTH_ICON)
```

Comment:
if we are setting document level properties, we should namespace them with `--cn` so that they don't have a chance of conflicting with the parent when running as an MFE

Reply:
Make sense👍

Reply:
FIxed